pkg/controllers/node/termination/terminator/events/events.go

```diff
@@ -34,6 +34,16 @@ func EvictPod(pod *v1.Pod) events.Event {
 	}
 }
 
+func DeletePod(pod *v1.Pod) events.Event {
+	return events.Event{
+		InvolvedObject: pod,
+		Type:           v1.EventTypeNormal,
+		Reason:         "Deleted",
+		Message:        fmt.Sprintf("Deleted pod regardless of PDBs and lifecycle hooks, %v seconds before node termination to accommodate its terminationGracePeriodSeconds", pod.Spec.TerminationGracePeriodSeconds),
```

Comment:
nit on following the convention of whatever you rename the event to, and minimizing the verbosity of the event.  I could see adding in the PDBs and lifecycle hooks bit if you think that's absolutely necessary, but I think saying "Forcibly" or "Killed" communicates the same semantic.
```suggestion
		Message:        fmt.Sprintf("Forcibly deleted pod %s seconds before node termination to accommodate its terminationGracePeriodSeconds", pod.Spec.TerminationGracePeriodSeconds),
```

Reply:
Given this is an event that I expect users to pay close attention to when understanding why their PDBs have been violated, I'd like to include all relevant details in the event message so they understand exactly why their pod was deleted without needing to consult cluster administrators.

This is what I've currently updated it to
```
"Deleting the pod to accommodate the terminationTime %v of the node. The pod was granted %v seconds of grace-period of its %v terminationGracePeriodSeconds. This bypasses the PDB of the pod and the do-not-disrupt annotation."
```

Reply:
Sounds good to me.

Reply:
Confirmed this level of verbosity is [acceptable](https://kubernetes.slack.com/archives/C04JW2J5J5P/p1719427319524189?thread_ts=1719426800.537739&cid=C04JW2J5J5P).